tests/LoginAttemptTest.php

```diff
@@ -79,31 +87,7 @@ public function testEmailVerification():void
 
             /** Try to log in with wrong password **/
             $response = $this->json('POST', '/auth/authenticate', ['email' => $userResult["email"], 'password' => 'wrong password']);
-            $response->assertResponseStatus(403);
```

Comment:
@jerazo should these tests have been removed?

Reply:
This is fine as if we do test the duration of elapse then the unit test will take much longer and might kill timeout.